association_test.go

```diff
@@ -792,7 +792,12 @@ func TestRelated(t *testing.T) {
 
 	var creditcard CreditCard
 	var user3 User
-	DB.First(&creditcard, "number = ?", "1234567890")
+	numberIs := "number = ?"
+	if isOra(DB) {
```

Comment:
I think that it is better to remove specific database judging. It should be implemented separately (I know oracle is very different).